api/README.md

```diff
@@ -36,3 +38,8 @@ Look for the second to last line in the console logs for the port. The line will
 
 2016-11-09 23:55:54.339  INFO 68013 --- [           main] s.b.c.e.t.TomcatEmbeddedServletContainer : Tomcat started on port(s): 8080 (http)
 
+### To Run Tests
+
+$ ./gradlew clean build
```

Comment:
These tests fail with an empty DB.

Reply:
you are right! I'll add an in memory db. But that's a bit more work than I can do at this moment. I'll do it with the global constant persistence trello task. I updated it with this info.

Reply:
Sorry, I edited my comment at the same time as you replied.

Reply:
`testGettingDevices` fails on first run but passes on subsequent runs because `testSendingNewReading` inserts into the readings table. 

`testSendingNewReading` reliably crashes during `POST` (line 182). The stack trace did not help me understand why.

Changing tests is outside the scope of this ticket. But if we expect tests fail in development, we should say so in the `README` -- until they are fixed.

Reply:
Tests don't fail for me!